### ISSUE TITLE
Sentinal Feat bugging out

### ISSUE THREAD

**Issue by kosocku** [SolastaMods/SolastaUnfinishedBusiness#2468]
I have 2 characters with the Sentinal feat.  When it comes to the OA stopping the movement of the target, there doesn't seem to be an issue.  However, I've now had 2 fights where the Barbarian (with Sentinal and Polearm Mastery), when making it's OA due to a target hitting an ally, will cause the target to make repeated attacks against said ally, letting you choose to make the OA every time, but not making it's OA.

The first time, the target (an Orc) made 3 attacks against my Wizard before the Barb actually made it's OA attack.

The second time (which is the combat saved in the linked file) it was a Giant Spider, and it ended up making multiple attacks against my Bard - with both my Soul Blade Paladin and Barbarian trying to make the reaction OAs - to the point of being able to kill the Bard before either of them actually made the attack.  When I finally did get to make an OA, I had to attack with the Pally and Pass on the attack with the Barb.

Then the 2nd spider attacked my Wizard, the Barb - who ended up not getting to make the attack on the first due to a smite - again tried to OA, but the game froze up solid and I had to Alt+F4 to get the game to close out.

Please note that I have made the Polearm Mastery OA (for coming within reach) numerous times with no issue, and the Sentinal feature of bringing the targets move speed to 0 works fine.

Game version: 1.4.33
UB Version 1.4.33.37
[UB Bug reports.zip](https://github.com/SolastaMods/SolastaUnfinishedBusiness/files/10736207/UB.Bug.reports.zip)

**Comment by ThyWoof** [comment]
I wasn’t able to reproduce with this save. Is there a good sequence that I could take on this battle?

**Comment by kosocku** [comment]
Hrm.  With this particular battle, it happened on the first turn for the spiders.  That being said, I've tried to recreate it after updating to 1.4.33.38 and have also been unable to reproduce, so I'm closing this issue.

Thanks for the time, and sorry about wasting it.

**Comment by kosocku** [comment]
It happened again today.  This time, however, it happened with the Paladin on it's OA instead of the Barbarian.  I've attached photos as well this time showing the chat log to show exactly what I'm talking about.  This time the Orc got multiple OAs when I moved my Bard away (with each giving the option for the Paladin to take an OA), and after the Bard finally moved, the Paladin too all of those OAs in a row.

Game 1.4.33
UB 1.4.33.38
[UB Bug reports.zip](https://github.com/SolastaMods/SolastaUnfinishedBusiness/files/10744048/UB.Bug.reports.zip)
